frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -384,10 +300,7 @@ class Dashboard extends Component {
                 {shouldRenderParametersWidgetInViewMode && (
                   <ParametersWidgetContainer
                     data-testid="dashboard-parameters-widget-container"
-                    ref={this.parametersWidgetRef}
-                    isNavbarOpen={isNavbarOpen}
```

Comment:
those props weren't used, ref is not needed as we do not calculate scrollTop anymore